### PR TITLE
Updated type definitions for email-templates for v3.1

### PR DESCRIPTION
**Pull request by blankstar85**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/niftylettuce/email-templates/blob/master/src/index.js
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by blankstar85** [comment]
Fixed Merge Conflict.

**Review by cyrilschumacher — APPROVED** [review]

**Comment by cyrilschumacher** [comment]
👍

File: types/email-templates/email-templates-tests.ts
Comment (DanielRosenwasser) [inline types/email-templates/email-templates-tests.ts]: Please keep this as `import EmailTemplates = require('email-templates')` given that the other file uses the `export =` form.

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]

File: types/email-templates/email-templates-tests.ts
Comment (blankstar85) [reply types/email-templates/email-templates-tests.ts]: done! thanks

**Comment by RyanCavanaugh** [comment]
Approved by a listed owner. PR ready to merge pending express review by a maintainer.

**Comment by RyanCavanaugh** [comment]
@DanielRosenwasser - Thanks for your review of this PR! Can you please look at the new code and update your review status if appropriate?

**Review by DanielRosenwasser — APPROVED** [review]

**Pull request merged**
